plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/ColumnWithAliasFormatterModule.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Singleton;
+
+public class ColumnWithAliasFormatterModule
+        extends AbstractModule
+{
+    @Override
+    public void configure()
+    {
+        bind(ColumnWithAliasFormatter.class).to(ColumnWithAliasFormatterFixed.class).in(Singleton.class);
```

Comment:
I can do that, but bear in mind that this would mean that I have a boilerplate Module in the test only, so that I can use the @Guice annotation. This is actually not that intrusive, as  I can make the test class inherit from AbstractModule and only add a configure() method to it. However, this would mean that the binding code would not be shared between production code and the test code, but will be two separate codebases. If that is the policy here and this is what you want to go with, I'm fine. Don't have a strong opinion here. Let me know what's your decision here.

Reply:
why does test code need a guice derived class? The `ColumnWithAliasFormatter` can be directly tested by calling it's public method instead of holding state in the class.

For the integration test you don't need to bind anything anyway since the query runner already has the TestingOraclePlugin (or something to that effect) installed which installs all the original modules anyway.

Reply:
Using DI in test just to create a single class is overkill. I'd agree for a class which needs other classes to be injected. We do use Guice to create those in tests but not for a class which has no dependencies.

What does DI buy us vs `new` for a class without any other injected classes?

Reply:
Scheduled a call with you on Tuesday 2023-09-05 to explain.

Reply:
So we agreed to keep existing module setup in production code (since thinking about using this in other connectors is out of scope for this PR).

In tests we'll switch to `new` on either `final` field in test class or initializing in `@BeforeClass` method.